node-repository/src/test/java/com/yahoo/vespa/hosted/provision/monitoring/MetricsReporterTest.java

```diff
@@ -193,7 +193,7 @@ private ApplicationId app(String tenant) {
         return Optional.empty();
     }
 
-    private static class TestMetric implements Metric {
+    public static class TestMetric implements Metric {
```

Comment:
Is it preferable to make a new implementation, rather than make this one public? I'd be able to make a really simple one ...

Reply:
It's fine imho